src/Common/Exception.h

```diff
@@ -194,43 +194,6 @@ class ErrnoException : public Exception
     const char * className() const noexcept override { return "DB::ErrnoException"; }
 };
 
-
-/// Special class of exceptions, used mostly in ParallelParsingInputFormat for
-/// more convenient calculation of problem line number.
-class ParsingException : public Exception
```

Comment:
It was used only in IRowInputFormat and ParallelParsingInputFormat for adding actual line number to the exception message. Now we don't need separate excpetion class for it